views/groceries.ejs

```diff
@@ -0,0 +1,23 @@
+<h2>Groceries List</h2>
+    <table id="groceries-table">
+      <tr id="groceries-table-header">
+        <th>Name</th>
+        <th>Added by</th>
+        <th>Status</th>
+        <th colspan="2"></th>
+      </tr>
+      <% if (groceries && groceries.length) { %>
+        <% groceries.forEach((product) => { %>
+          <tr>
+            <td><%= product.name %></td>
+            <td><%= product.addedby %></td>
+            <td><%= product.status %></td>
+            <a href="/groceries/edit/<%= product.id %>"><button type="button">edit</button></a>
+            <form method="POST" action="/groceries/delete/<%= product.id %>" style="display: inline">
+                <button>delete</button>
+                <input type="hidden" name="_csrf" value="<%= _csrf %>" />
+            </form>
+          </tr>
+        <% }) %>
+      <% } %>
+    </table>
```

Comment:
Add a button after this line to add a new product.
Example:
```
<a href="/groceries/new">
      <button type="button">create new product</button>
      </a>
```
Add in the last line the include to the footer.ejs page.
Example:
```
<%- include("partials/footer.ejs") %>
```